qmanager/modules/qmanager.cpp

```diff
@@ -100,7 +102,9 @@ static int process_config_file (std::shared_ptr<qmanager_ctx_t> &ctx)
     optmgr_kv_t<qmanager_opts_t> opts_store;
     std::string info_str = "";
     json_object_foreach (conf, k, v) {
-        std::string value = json_string_value (v);
+        std::string value = json_dumps (v, JSON_ENCODE_ANY|JSON_COMPACT);
+        if (json_typeof (v) == JSON_STRING)
+            value = value.substr (1, value.length () - 2);
```

Comment:
I assume this is here to strip the quotes off the ends of the string?  For some reason this feels like it could be fragile, I'm not sure though.  Any opposition to just grabbing the string value via the jansson API?

```suggestion
        if (json_typeof (v) == JSON_STRING)
            value = json_string_value (v);
```

If you want to keep the `substr`, can you leave a short comment as to the purpose?

Reply:
That was my initial code but I changed it so that one can add non string values within `sched-fluxion-qmanager.toml` like our dropped support for `resource-recovery-on-load`

```
resource-recovery-on-load = true
```

As opposed to 

```
resource-recovery-on-load = "true"
```

can be supported.